tests/utils/getpackagenames.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint mocha:true */
+
+'use strict';
+
+const getPackageNames = require( '../../lib/utils/getpackagenames' );
+const expect = require( 'chai' ).expect;
+
+describe( 'utils', () => {
+	describe( 'getPackageNames()', () => {
```

Comment:
Can you pass multiple rules in a glob string – like – `a,b,c`? Please check if minimatch supports it this way and if so, let's have a test for it. If not, let's ignore this as it's not worth too much time.